src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return (firstNum) => {
+    counter++;
+
+    let num = firstNum;
+
+    return (secondNum) => {
```

Comment:
```suggestion
  return (a) => (b) => {
    counter++;
    //and add some checking
    return a + b; 
```
I think you can replace with shorter construction